lib/commanded/event_store/adapters/spear/subscription.ex

```diff
@@ -31,6 +31,10 @@ defmodule Commanded.EventStore.Adapters.Spear.Subscription do
   Start a process to create and connect a persistent connection to the Event Store
   """
   def start_link(conn, stream, subscription_name, subscriber, serializer, opts) do
+    if Keyword.get(opts, :partition_by) do
+      Logger.warn("Spear does not support partitioning. The partition_by option will be ignored.")
+    end
```

Comment:
I would prefer to see an error here than a warning log IMO

Reply:
@yordis thanks for the feedback. Do you mean changing the log level from warning to error?

Reply:
I meant `raise, "dont start the process since it will behave unexpectedly."`

Reply:
makes sense, changed it in: https://github.com/fabriziosestito/commanded-spear-adapter/pull/22